docs/general/attributes.md

```diff
@@ -412,9 +412,9 @@ These attributes may be used for any operation with an authenticated and/or auth
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
-These attributes describe the authenticated user driving the user agent making requests to the instrumented
+`enduser.id` describe the authenticated user driving the user agent making requests to the instrumented
 system. It is expected this information would be propagated unchanged from node-to-node within the system
-using the Baggage mechanism. These attributes should not be used to record system-to-system
+using the Baggage mechanism. This attribute should not be used to record system-to-system
```

Comment:
is this expected? would client SDK populate them on the baggage? if not, let's just remove it. This was deprecated before, we should document the new behavior.

Reply:
removed.